reactive/src/Components/NewBeer.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Component } from "react";
+import Navbar from "./Navbar/Navbar";
+import axios from "axios";
+
+class NewBeer extends Component {
+  constructor (props) {
+    super (props);
+    this.state = { name:"", tagline:"", description:"", first_brewed:"", brewers_tips:"", attenuation_level:0, contributed_by:"" }
+  }
+  handleFormSubmit = (event) => {
+    event.preventDefault();
+    const name = this.state.name;
+    const tagline = this.state.tagline;
+    const description = this.state.description;
+    const first_brewed = this.state.first_brewed;
+    const brewers_tips = this.state.brewers_tips;
+    const attenuation_level = this.state.attenuation_level;
+    const contributed_by = this.state.contributed_by;
```

Comment:
Hello Béatrice, voici quelques commentaires qui vont suivre.
Tu peux remplacer tout ce bout ce code par : 
const {
      name,
      tagline,
      description,
      first_brewed,
      brewers_tips,
      attenuation_level,
      contributed_by
    } = this.state;